frontend/src/categoriesPage/ManageCategories.tsx

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Container, Col, Row } from "react-bootstrap";
+import Category from "./Category";
+import AddNewCategoryModal from "./AddNewCategoryModal";
+
+export interface ICategory {
+    id: number
+    name: string
+    color: string
+}
+
+function ManageCategories() {
+
+    const [categories, setCategories] = useState<ICategory[]>([])
+
+    useEffect(() => {
+        axios.get<ICategory[]>("http://localhost:8080/todoCategories").then(response => {
+            setCategories(response.data)
+        })
+    }, [])
+
+    return (
+        <Container>
+            <Row >
+                {categories.map(category =>
+                    <Col className="d-flex flex-wrap" xs="12" sm="6" md="3">
```

Comment:
Col needs a key property, see sonarlint: Missing "key" prop for element in iteratorsonarlint(typescript:S6477)

Reply:
okay boss